budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/persistence/manualRecord/ManualRecordEntity.java

```diff
@@ -0,0 +1,89 @@
+package org.wickedsource.budgeteer.persistence.manualRecord;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.wickedsource.budgeteer.persistence.budget.BudgetEntity;
+
+import javax.persistence.*;
+import java.util.Calendar;
+import java.util.Date;
+
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ManualRecordEntity {
+    @Id
+    @GeneratedValue
+    private long id;
+
+    @Column(nullable = false, length = 255)
+    private String description;
+
+    @Column(nullable = false)
+    private int cents;
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "BUDGET_ID")
+    private BudgetEntity budget;
+
+    @Temporal(TemporalType.DATE)
+    @Column(name="RECORD_DATE", nullable = false)
+    private Date date;
```

Comment:
Records have dates (the date they were added). Maybe they should also have another date, like the date said action actually happend (maybe it could be optional). Then the user could see two columns in the table - Record Date and Date Added.